### ISSUE TITLE
ColorVariable linter incorrectly flags overflow and cursor properties.

### ISSUE THREAD

**Issue by olliebennett** [causes/scss-lint#357]
Using latest master (tested @ 10f2725eaef871cfe454d06e5f07372ec7818826), I see a couple of properties being incorrectly flagged by the new `ColorVariable` linter.

It has happened from the first introduction of this linter (in e72cb231906be28dbe4c523706c2f0594623e6dd), although the error message has been tweaked since then. Apologies for not catching it earlier.

```scss
.test-class {
  overflow: hidden;
  cursor: default;
}
```
is linted with the following output:
```
stylesheets/test.scss:2 [W] ColorVariable: Color literals like `hidden` should only be used in variable declarations; they should be referred to via variable everywhere else.
stylesheets/test.scss:3 [W] ColorVariable: Color literals like `default` should only be used in variable declarations; they should be referred to via variable everywhere else.
```

**Comment by sds** [comment]
Thanks for the report, @olliebennett!

This was a regression caused by a refactor in 8637aa633. I've fixed it in da2961029ae43ba69d965abd3f45a5ea0ea1117d.

**Comment by olliebennett** [comment]
Yep - working now. Awesome job! :+1: Thanks a lot
